gradle/script-git-version.gradle

```diff
@@ -0,0 +1,18 @@
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath("org.ajoberstar.grgit:grgit-gradle:4.1.1")
+    }
+}
+
+import org.ajoberstar.grgit.Grgit
+
+ext {
+    println("projectDir = "+projectDir)
+    git = Grgit.open(currentDir: projectDir)
+    def tagSize = git.tag.list().size()
+    gitVersionCode = tagSize+1
+    gitVersionName = "0.1.$tagSize".toString()
```

Comment:
So just number of tags + 1? Do we filter release tags somehow?

Reply:
Right, it will increase, but since all the tags are counted and not just release ones - I assume numbers will be pretty random, e.g. if number of tags is 1, 5, 6, 9 the version names would be `0.1.1`,`0.1.5`, `0.1.6`, `0.1.9`, not that informative IMO. Could we filter release tags somehow, since we should know the release tag naming pattern?

Reply:
yeah correct. it could be pretty random. but the rave team doesn't care about version, they check release notes for testing. I will try if i can filter with naming pattern. otherwise, we will keep it like this for now. :)

Reply:
as an option we can use date+[time] for this version
`0.1.20222906`

Reply:
done using `git.tag.list().findAll { (it.name =~ "android-v*") **}.size()`